scripts/deploy.sh

```
@@ -3,8 +3,14 @@ set -e
 BRANCH_NAME=${BRANCH_NAME:-$(git rev-parse --abbrev-ref HEAD)}
 # Sanitize branch name for partykit preview (must match /^[a-z0-9_-]+$/)
 BRANCH_NAME=$(echo "$BRANCH_NAME" | tr '/' '-' | tr '[:upper:]' '[:lower:]')
+MAX_PREVIEW_BRANCH_NAME_LENGTH=25
 COMMIT_HASH=${COMMIT_HASH:-$(git rev-parse --short HEAD)}
 
+if [ "$BRANCH_NAME" != "main" ] && [ ${`#BRANCH_NAME`} -gt $MAX_PREVIEW_BRANCH_NAME_LENGTH ]; then
```

Shellcheck (0.11.0)

[error] To expand via indirection, use arrays, ${!name} or (for sh only) eval.

(SC2082)